lib/SparqlGenerator.js

```diff
@@ -1,5 +1,7 @@
 var XSD_INTEGER = 'http://www.w3.org/2001/XMLSchema#integer';
 
+var termToString = require("rdf-string").termToString;
```

Comment:
Should not be needed, as only variables are stringified. This should be sufficient: `'?' + term.value`

Reply:
One of the functions using termToString is also called with NamedNode instances. Is this enough reason to keep the module in?

Reply:
Probably not, as termToString will emit those as `"url"`, while SPARQL requires `<url>`.

Reply:
It gets called with BlankNode instances as well so I would suggest to keep it. What do you think?

Reply:
Bnodes are quite simple as well, that requires just prefixing the value with `"_:"`.